sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md

```diff
@@ -18,6 +18,11 @@
 the telemetry offline for retrying at a later time.
 ([#38832](https://github.com/Azure/azure-sdk-for-net/pull/38832))
 
+* Fixed an issue where `OriginalFormat` persisted in TraceTelemetry properties
+  with IncludeFormattedMessage enabled in OpenTelemetry LoggerProvider. This fix
```

Comment:
`IncludeFormattedMessage` set to `true` on [OpenTelemetryLoggerOptions](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/Logs/ILogger/OpenTelemetryLoggerOptions.cs)